e2e_tests/tests/experiment/test_custom_searcher.py

```diff
@@ -0,0 +1,580 @@
+import logging
+import subprocess
+import tempfile
+import time
+from pathlib import Path
```

Comment:
[google style guide for imports](https://google.github.io/styleguide/pyguide.html#22-imports)